### ISSUE TITLE
Kong startup throws  Symbol not found: _fastsearch in mac osx

### ISSUE THREAD

**Issue by tamizhgeek** [Mashape/kong#213]
Hi,

I tried to setup a kong dev env using the instructions here : https://github.com/Mashape/kong/blob/master/README.md

While doing [sudo] make dev, I am getting this error
```
bin/kong migrations -c kong_DEVELOPMENT.yml up
lua: error loading module 'stringy' from file '/usr/local/lib/lua/5.1/stringy.so':
	dlopen(/usr/local/lib/lua/5.1/stringy.so, 2): Symbol not found: _fastsearch
  Referenced from: /usr/local/lib/lua/5.1/stringy.so
  Expected in: flat namespace
 in /usr/local/lib/lua/5.1/stringy.so
stack traceback:
	[C]: ?
	[C]: in function 'require'
	./kong/dao/cassandra/factory.lua:10: in main chunk
	[C]: in function 'require'
	./kong/tools/io.lua:95: in function 'load_configuration_and_dao'
	./kong/cli/migrations.lua:27: in main chunk
	[C]: in function 'require'
	bin/kong:53: in main chunk
	[C]: ?
make: *** [dev] Error 1
```

OS: Mac OSX Yosemite
Lua : 5.1
Luarocks: 2.2.1
Kong: Latest master

Google didn't help. Tried using lua 5.2, but that was throwing incompatible errors during make install itself. Any help is highly appreciated!

**Comment by tamizhgeek** [comment]
Some additional information: 

Kong depends on multipart, which inturn depends on stringy 0.2-1
Kong itself also has a direct dependency on stringy 0.4-1. 

I uninstalled the stringy 0.4-1 rock manually, and then things started to work just fine.

**Comment by thefosk** [comment]
@tamizhgeek that is very weird - I am running on OS X Yosemite too, Lua 5.1.4, Luarocks 2.2.2 and never seen that error before - I need to investigate

**Comment by tamizhgeek** [comment]
@thefosk What is the stringy version you have? Also, why is there a explicit dependency of stringy tied to version 0.4 in the rockspec? I removed it, and things are working just fine.

**Comment by thefosk** [comment]
I am using stringy `0.4-1` which should be the proper version to have as version `0.2-1` has some serious bugs on some functions and will be soon removed from Kong.

`0.4-1` is also the latest version of stringy.

**Comment by thibaultCha** [comment]
Please reopen if you still experience this, but it should be fixed in `0.2.1`!
